src/Authentication/Controllers/AuthenticationController.cs

```diff
@@ -355,7 +355,7 @@ private async Task<ActionResult> AuthenticateAltinnStudioToken(string originalTo
                     ValidateAudience = false,
                     RequireExpirationTime = true,
                     ValidateLifetime = true,
-                    ClockSkew = TimeSpan.Zero
+                    ClockSkew = new TimeSpan(0, 0, 10)
```

Comment:
TimeSpan.FromSeconds is so much easier to read/understand ~10 seconds down the line xD

(this applies to all of them, just leaving one comment though)

Reply:
Done